src/assets/scripts/client/navigationLibrary/FixCollection.js

```diff
@@ -100,6 +100,24 @@ class FixCollection extends BaseCollection {
         return fixModel.relativePosition;
     }
 
+    /**
+     * Returns the pronunciation of the fix name if it exists, otherwise returns the input name
+     *
+     * @for FixCollection
+     * @method getFixSpokenName
+     * @param fixName {string}
+     * @return {string}
+     */
+    getFixSpokenName(fixName) {
```

Comment:
I find it hard to think of a case where we would have the fix name, but not the FixModel itself. Should we not just achieve this with `FixModel.getSpokenName()`?

Is this maybe just for the readbacks, like with the `pd` command? If so and the fix model isn't readily accessible (or inconvenient), this is fine. Just checking since I wasn't expecting to see it here.

Reply:
There are two reasons why I chose to do it this way.

1. Easier use of the function for evaluating the spoken name directly in readback strings, without having to look up if the actual fix exists or not. If it doesn't exist the input is read back.

2. The readback string is constructed in a few different places, with access to different object types. I'm on mobile right now, but I recall that somewhere it had a FixModel, in most places a WaypointModel, and I think there was a HoldModel or something at one place. These other types of models don't have an easy way to access the underlying FixModel, they have just copied the fix name. So we can't access the FixModel without searching for it in the FixCollection anyway. So I decided this solution would need less code and lookups from different places.

I prefer to leave this as it is, if you don't have very strong feelings about doing it another way.

Reply:
Okay, that all makes sense. Fine as is.